tools/WinObjC.Packaging/WinObjC.Packageable.props

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGABLE PROPS
+  Description: The property definitions specific to a WinObjC project intended to be packaged
+  For Use By: A VcxProj that can be packaged into a NuGet
+-->
+
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <!-- NuGet specific properties -->
+    <IncludeOutputsInPackage>false</IncludeOutputsInPackage>
+    <IncludeContentInPackage>false</IncludeContentInPackage>
+
+    <!-- WinObjC.Packaging specific properties -->
+    <PresetPackageContents Condition="'$(PresetPackageContents)' == ''">true</PresetPackageContents>
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <_MainProjectOutput>
+      <IsPackable>$(IsPackable)</IsPackable>
```

Comment:
>IsPackable [](start = 7, length = 10)

Yeah this one (my other comment about keying off of metadata instead of extension)